.evergreen/config.yml

```diff
@@ -708,43 +708,47 @@ axes:
       - id: "latest"
         display_name: "MongoDB latest"
         variables:
-           VERSION: "latest"
+           MONGODB_VERSION: "latest"
+      - id: "rapid"
+        display_name: "MongoDB rapid"
+        variables:
+           MONGODB_VERSION: "rapid"
       - id: "6.0"
         display_name: "MongoDB 6.0"
         variables:
-           VERSION: "6.0"
+           MONGODB_VERSION: "6.0"
       - id: "5.0"
         display_name: "MongoDB 5.0"
         variables:
-           VERSION: "5.0"
+           MONGODB_VERSION: "5.0"
       - id: "4.4"
         display_name: "MongoDB 4.4"
         variables:
-           VERSION: "4.4"
+           MONGODB_VERSION: "4.4"
       - id: "4.2"
         display_name: "MongoDB 4.2"
         variables:
-           VERSION: "4.2"
+           MONGODB_VERSION: "4.2"
       - id: "4.0"
         display_name: "MongoDB 4.0"
         variables:
-           VERSION: "4.0"
+           MONGODB_VERSION: "4.0"
       - id: "3.6"
         display_name: "MongoDB 3.6"
         variables:
-           VERSION: "3.6"
+           MONGODB_VERSION: "3.6"
 
   - id: mongodb-edge-versions
     display_name: MongoDB Version
     values:
       - id: "latest-stable"
         display_name: "MongoDB 6.0"
         variables:
-          VERSION: "6.0"
+          MONGODB_VERSION: "6.0"
```

Comment:
As I mentioned in [Slack](https://mongodb.slack.com/archives/C72LB5RPV/p1680670614828379?thread_ts=1680589736.626789&cid=C72LB5RPV):

> On a related note, I'm thinking it probably makes sense to use the rapid releases for the "latest-stable" target in our MongoDB version axis. This would conceivably remove the need to update that target in the future (last bump was from 5.0 to 6.0 IIRC).

Thoughts?

Reply:
Sounds good, "rapid" is indeed the same now.

Reply:
I decided to leave this as-is. See: https://github.com/mongodb/mongo-php-driver/pull/1408#discussion_r1162418220